src/js/index.js

```diff
@@ -0,0 +1,81 @@
+import {INITAIL_VALUE, ALERT_MSG_OVER_THREE_NUMBER, ALERT_MSG_WITH_OUT_NUMBER} from './constant.js'
+
+const $display = document.querySelector('#total');
+const $allClear = document.querySelector('.modifier');
+const $operations = document.querySelector('.operations');
+const $digits = document.querySelector('.digits');
+
+let firstNumber = '';
+let operation = '';
+let secondNumber = '';
+let result = 0;
+
+const allClearClick = ()=>{
+    firstNumber = '';
+    secondNumber = '';
+    operation = '';
+    result = 0;
+    display(true);
+}
+
+const digitClick = (event) => {
+    if(operation == ''){
+        if(firstNumber.toString().length >= 3) {
+            alert('숫자는 세 자리까지만 입력 가능합니다!')
+            return;
+        }
+        firstNumber = parseInt(firstNumber + event.target.innerText);
+        display();
+    }else{
+        if(secondNumber.toString().length >= 3) {
+            alert(ALERT_MSG_OVER_THREE_NUMBER)
+            return;
+        }
+        secondNumber = parseInt(secondNumber + event.target.innerText);
+        display();
+    }
+}
```

Comment:
```suggestion
const digitClick = (event) => {
    if(operation == ''){
        if(firstNumber.toString().length >= 3) {
            alert('숫자는 세 자리까지만 입력 가능합니다!')
            return;
        }
        firstNumber = parseInt(firstNumber + event.target.innerText);
    }else{
        if(secondNumber.toString().length >= 3) {
            alert(ALERT_MSG_OVER_THREE_NUMBER)
            return;
        }
        secondNumber = parseInt(secondNumber + event.target.innerText);
    }
    display();
}
```

`display()`를 if-else 문 외부로 빼서 중복되는 코드를 줄여보는건 어떨까요?

Reply:
오호 그러네요 !! 감사합니다!  :+1: :+1: :+1: